be/src/formats/parquet/group_reader.cpp

```diff
@@ -557,24 +553,51 @@ Status GroupReader::DictFilterContext::rewrite_conjunct_ctxs_to_predicates(
             return Status::OK();
         }
 
-        // get dict codes
+        // ---------
+        // get dict codes according to dict values.
+        dict_value_column = dict_value_chunk->get_column_by_slot_id(slot_id);
```

Comment:
Why get it again? it is the same as LOC:542

Reply:
Because I'm not sure if the column has been deleted and created during this function.
```
        RETURN_IF_ERROR(ExecNode::eval_conjuncts(_conjunct_ctxs_by_slot[slot_id], dict_value_chunk.get()));
```
Maybe there is no need to get it again.

Reply:
Yeah, no need to get it.